pkg/plugins/policies/core/rules/rules_test.go

```diff
@@ -241,37 +241,128 @@ var _ = Describe("Rules", func() {
 			func(given testCase) {
 				Expect(given.s1.Intersect(given.s2)).To(Equal(given.intersect))
 			},
-			Entry("entry 1", testCase{
+			Entry("positive, same key and value", testCase{
+				s1: []core_rules.Tag{
+					{Key: "service", Value: "frontend"},
+				},
+				s2: []core_rules.Tag{
+					{Key: "service", Value: "frontend"},
+				},
+				intersect: true,
+			}),
+			Entry("positive, same key, different value", testCase{
```

Comment:
what does positive mean?

Reply:
It's the opposite of negation,
We have the structure:

```text
type Tag struct {
	Key   string
	Value string
	Not   bool
}
```

From the codes and comments, I saw some places define it as `negation` with `Tag.Not==true`. 
So, I say it `positive` with `Tag.Not==false`

Reply:
This concept comes from the original code: https://github.com/kumahq/kuma/blob/86593067f05038a1c7f76446a177538f3efff00f/pkg/plugins/policies/core/rules/rules.go#L238-L247